manager/controllers/app/multicluster/multi_cluster_manager.go

```diff
@@ -0,0 +1,75 @@
+package multicluster
```

Comment:
I suggest to start moving things to the root level `/pkg`, starting with new stuff like multicluster.

Reply:
I don't mind, @froesef do you see a reason not to do that?